pkg/webhook/utils/service.go

```diff
@@ -32,6 +36,18 @@ func MutateLBService(new, old *corev1.Service) error {
 		new.Spec.HealthCheckNodePort == 0 {
 		new.Spec.HealthCheckNodePort = old.Spec.HealthCheckNodePort
 	}
+}
 
-	return nil
+// MutateAnnotation mutates annotation of LoadBalancer type service
+func MutateAnnotation(new, old *corev1.Service) {
+	if new.Spec.Type != corev1.ServiceTypeLoadBalancer {
+		return
+	}
+
+	if new.Annotations == nil {
+		new.Annotations = make(map[string]string)
+	}
+	if _, exist := new.Annotations[AlicloudLoadBalancerSpecAnnotationKey]; !exist {
+		new.Annotations[AlicloudLoadBalancerSpecAnnotationKey] = AlicloudLoadBalancerSmallSpec
+	}
```

Comment:
You could use `metav1.SetMetaDataAnnotation` like in https://github.com/gardener/gardener/blob/54311dcd102a42d0e851bc2a6d69797f4869ba1a/extensions/pkg/webhook/controlplane/checksums.go#L38

Reply:
done